### DISCUSSION TITLE
I wish I could upload static assets from the browser IDE

### DISCUSSION THREAD

**Discussion by pbiggar** [darklang/dark#3978] [Feedback]
A user says:
>The idea is cool and the online IDE is neat, but the text editor is uncomfortable when you're deleting/editing parameters and I wish I could upload static assets from the browser IDE

**Comment by zsombro** [comment]
Hey, this was me! I really appreciate that you care so much about all the feedback, I still think Dark has an intriguing core concept and I hope to see it grow!
